pylabs/Compat.py

```diff
@@ -144,7 +144,7 @@ def copyDirTree(self,src,dst):
 
     root = os.environ.get('TEST_HOME')
     if root is None:
-        root = os.environ['HOME']
+        raise Exception("TEST_HOME is undefined")
```

Comment:
impact on travis/jenkins?

Reply:
I have no idea, but after cleaning my home directory several times I believe this is the right thing to do.